ninja-servlet/src/main/java/ninja/servlet/ContextImpl.java

```diff
@@ -221,7 +260,22 @@ public Integer getParameterAsInteger(String key, Integer defaultValue) {
 
     @Override
     public Map<String, String[]> getParameters() {
-        return httpServletRequest.getParameterMap();
+        // create new map with query params
+        Map<String, String[]> map = new HashMap<>(httpServletRequest.getParameterMap());
+
+        // add params of multipart request
+        for (Entry<String, List<String>> e : multipartParams.entrySet()) {
```

Comment:
also please remove the space between e and : => please do that everywhere in the PR... Nothing personal.. it's just how the other code of Ninja is formatted...